week3/ml_decision_tree_sample/unit_tests/test_code.py

```diff
@@ -0,0 +1,29 @@
+import sys, os
+import pandas.api.types as ptypes
+import pytest
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname('ml_decision_tree_sample'), '..')))
```

Comment:
remove this, please use PYTHONPATH instead

Reply:
It will not see modules otherwise

Reply:
if you use PYTHONPATH, it would see it

Reply:
`PYTHONPATH` is not work for this case
I added like this: <`export PYTHONPATH="/home/lubomir/Desktop/ML_in_production_home_works/week3"`>
I change patch to module  in python file to `from week3.ml_decision_tree_sample.predictor import model_for_predict`
and called this python file from root directory and in this case i receive this error  `ModuleNotFoundError: No module named 'week3'`